### DISCUSSION TITLE
I dont know what the FUCK im doing

### DISCUSSION THREAD

**Discussion by toma966** [UltimaHoarder/UltimaScraper#1542] [Q&A]
Is this website meant for computer science majors and coders only?
Nothing about the guide makes sense to me. My compters terminals/ python and the start_ofd file keep giving me error messages. Can someone please make a realistic step by step guide? When i go on onlyfans can u explain how to get the cookies. Im so confused. My subscription ends on the 11th so I have 3 days. Thanks to whoever heps

**Comment by LunarPenguin** [comment]
1. download and install python 3.9
2. download the latest version of the script and unzip to a location you want the script to be at.
3.  go to the folder from step 2
4. press <kbd>shift</kbd> and right click on a empty part in the file explorer from step 3 and click `open powershell here` or something similar.
5. In the new powershell window type `python3 -V` or `py -3 -V`. if you get the python version returned like this `Python 3.9.x` you're good to go.
6. type `python3 -m pip install -U -r requirements.txt` or `py -3 -m pip install -U -r requirements.txt` to install the dependencies needed.
7. if step 6 completed successfully type `py -3 start_ofd.py` or `python3 start_ofd.py` to start the scraper for the first time, this creates the needed config files.

 to get the cookies: 
1. go to onlyfans and login if you have not already.
2. open the developer tools for the browser you are using, for most browsers you can press <kbd>f12</kbd> or right click and choose `inspect element`. after that you can use the explanation in the `readme` file.

to start scraping follow steps 3,4 and 7 to open a new powershell window or reuse the existing window if that is still open. after that follow the prompts on the screen.

i hope this helps please let us know if you have any errors.

**Reply by OnyxOperator** [reply]
Everything is good until the last step where I get "[Errno 2] No such file or directory" when trying to run "py -3 start_ofd.py"

Nothing in FAQ, Discussions, or Issues shows anything regarding this error so I'm at a loss

**Reply by LunarPenguin** [reply]
are you in the right folder?, do you perhaps have a screenshot?

**Reply by OnyxOperator** [reply]
![folder](https://user-images.githubusercontent.com/88251924/130544172-3eb44e1c-9dfc-4f4e-9346-a7e1f2ae11db.png)
This is the folder straight from the site, unless there was a location preference or some other painfully obvious step I'm missing

**Reply by LunarPenguin** [reply]
I think you did something wrong cause its supposed to look like this (give or take a few files and folder):
![image](https://user-images.githubusercontent.com/35904804/130661486-24133ac5-315b-4bf6-824d-71c28363491c.png)

Installation is wherever you want it to be. from here just open up the powershell window and run `py -3 start_ofd.py` or your equivalent command but it seems your also running an older version (3.7.2). [this](https://github.com/DIGITALCRIMINAL/OnlyFans/releases/tag/v7.4.1) is the latest version

**Reply by OnyxOperator** [reply]
yeah I got it to run with the latest version, thank you!

**Reply by LunarPenguin** [reply]
> yeah I got it to run with the latest version, thank you!

That's great to hear

**Comment by fusti80** [comment]
Buenas, soy muy neófito en la materia, me he quedado atascado en el punto 6. 

Me requiere la actualización del comando (- m) dentro de la aplicacion de Python, en el apartado (python.exe) pero no encuentro este apartado por ningun lado

**Reply by LunarPenguin** [reply]
if you have downloaded the repo you will find within it a file called requirements.txt. with the command at point six u can use that file to install all the required libraries. Do you perhaps have a screenshot, log or video of what is going wrong? that might help troubleshooting.

**Reply by NIFTY-SKIDDIE** [reply]
@LunarPenguin when I start the program, i just see the "press enter to continue" and when i press enter nothing happens. any idea? got all the requirements and dependencies
